drivers/hud_spm_report/app/models/hud_spm_report/fy2023/return.rb

```diff
@@ -22,6 +22,10 @@ def project_id
       [exit_enrollment, return_enrollment].detect(&:present?)&.enrollment&.project&.id
     end
 
+    def data_source_id
+      [exit_enrollment, return_enrollment].detect(&:present?)&.enrollment&.data_source_id
```

Comment:
Nothing wrong with this, but I think it might be slightly more readable as:
```suggestion
      [
        exit_enrollment&.enrollment&.data_source_id,           
        return_enrollment&.enrollment&.data_source_id,
       ].detect(&:present?)
```

Reply:
updated in [this commit](https://github.com/greenriver/hmis-warehouse/pull/5576/commits/920b71ab46a025fb97c4e27c64e4657367a7f5f5)